src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -72,104 +66,85 @@ protected void initialize() {
 
             @Override
             protected boolean isFinished() {
-                // Make this return true when this Command no longer needs to run execute()
                 return false;
             }
         };
     }
 
     public Command turnOnGreen(Integer... banks) {
-        // The command is named "Sample Command" and requires this subsystem.
-        return new Command("Turn on the color known as verde", this) {
+        return new Command("Green On", this) {
 
             @Override
             protected void execute() {
-                // Called repeatedly when this Command is scheduled to run
                 for (Integer currentBank : banks) {
                     ldrive_can.SetColor(currentBank, Color.GREEN, 1.0);
-
                 }
                 ldrive_can.Update();
             }
 
             @Override
             protected boolean isFinished() {
-                // Make this return true when this Command no longer needs to run execute()
                 return false;
             }
         };
     }
 
     public Command turnOnRed(Integer... banks) {
-        return new Command("Turn on the color known as rojo", this) {
+        return new Command("Red On", this) {
 
             @Override
             protected void execute() {
-                // Called repeatedly when this Command is scheduled to run
                 for (Integer currentBank : banks) {
                     ldrive_can.SetColor(currentBank, Color.BLUE, 1.0);
```

Comment:
This says it sets the color to `Color.BLUE`...

Reply:
Yes?